tools/importer-rest-api-specs/components/parser/parser.go

```diff
@@ -92,6 +92,11 @@ func (d *SwaggerDefinition) simplifyOperationNamesForResource(resource models.Az
 	output := make(map[string]models.OperationDetails)
 	for key, value := range resource.Operations {
 		updatedKey := key[len(resourceNameLower):]
+		// trim off any spurious `s` at the start, to handle tags with singular vs plural
+		if strings.HasPrefix(updatedKey, "s") {
+			updatedKey = updatedKey[1:]
+		}
```

Comment:
Comment implies tags is the only instance where we might have a spurious 's', in which case I think code should be updated to address only that case (or comment updated/clarified)

Reply:
tags here == Swagger Tags not the literal value `tags` - but I'll update that to clarify that